packages/core/src/messagingProtocols/proteus/ProteusService/CryptoClient/CryptoboxWrapper.ts

```diff
@@ -20,9 +20,10 @@
 import {PreKey} from '@wireapp/api-client/lib/auth';
 
 import {Cryptobox} from '@wireapp/cryptobox';
+import {keys as ProteusKeys} from '@wireapp/proteus';
```

Comment:
Doesn't coreCrypto expose this value? I think at some point we would like to remove `@wirapp/proteus` at all

Reply:
Oh wait, this is the cryptobox wrapper, all good you can ignore my comment :)

Reply:
I don't think we want to use coreCrypto type in cryptobox wrapper, do we? So maybe let me just revert it and keep the value on our side for cryptobox?